flaml/default/suggest.py

```diff
@@ -55,7 +62,9 @@ def suggest_config(task, X, y, estimator_or_predictor, location=None, k=None):
     )
     assert predictor["version"] == "default"
     prep = predictor["preprocessing"]
-    feature = meta_feature(task, X, y)
+    feature = meta_feature(
+        task, estimator_class=estimator_or_predictor, X_train=X, y_train=y

```

Comment:
`estimator_or_predictor` can be either a string or a dict. When it's a str, it can be "all". Please take that into consideration and make changes as needed.
For example, when it's a dict, shall we include the meta feature names in the dict? That requires changing the predictor .json schema and add a key "meta_feature_names".
Considering this, maybe we shouldn't pass estimator_or_predictor. Maybe we should pass the "meta_feature_names" field in `predictor` to `meta_feature`.

Reply:
?